db/models/comments.model.js

```diff
@@ -25,3 +25,13 @@ exports.selectCommentById = (comment_id) => {
         return Promise.reject({ status: 404, msg: "not found" });
     });
 };
+
+exports.selectArticleCommentCount = (article_id) => {
```

Comment:
While having a separate function to get the comment count will work, it is possible (and maybe preferable) to do this with one query in the `selectArticleById` function, using a JOIN to add a column to the article query.

Reply:
Thanks.